src/org/mozilla/javascript/NativeJSON.java

```diff
@@ -294,10 +294,17 @@ private static Object str(Object key, Scriptable holder, StringifyState state) {
             value = getProperty(holder, ((Number) key).intValue());
         }
 
+        if (ScriptRuntime.isSymbol(value)) return Undefined.instance;
```

Comment:
Is this correct, from an EcmaScript pov, that this is here, before the toJSON call or the call to the replacer (if provided)?

Reply:
@p-bakker you are right here - update is on the way. Many thanks for the review.